app/src/main/java/org/mozilla/fenix/customtabs/CustomTabToolbarMenu.kt

```diff
@@ -148,7 +148,7 @@ class CustomTabToolbarMenu(
             SimpleBrowserMenuItem(
                 {
                     val appName = context.getString(R.string.app_name)
-                    context.getString(R.string.browser_menu_powered_by, appName).toUpperCase()
+                    context.getString(R.string.browser_menu_powered_by2, appName).toUpperCase()
```

Comment:
Won't this make the notification all caps? If so, why is browser_menu_powered_by2 necessary?

Reply:
The "2" is for a not-all-caps version of the string.